lib/adapters/file/write.js

```diff
@@ -0,0 +1,85 @@
+var _ = require('underscore');
+var Juttle = require('../../runtime/index').Juttle;
+var fs = require('fs');
+
+var Write = Juttle.proc.sink.extend({
+    procName: 'write-file',
+    initialize: function(options, params) {
+        var allowed_options = ['file', 'limit'];
+        var unknown = _.difference(_.keys(options), allowed_options);
+        if (unknown.length > 0) {
+            throw this.compile_error('RT-UNKNOWN-OPTION-ERROR', {
+                proc: 'write file',
+                option: unknown[0]
+            });
+        }
+
+        if (!_.has(options, 'file')) {
+            throw this.compile_error('RT-MISSING-OPTION-ERROR', {
+                proc: 'write file',
+                option: "file"
+            });
+        }
+
+        this.limit = options.limit ? options.limit : 100000;
```

Comment:
It is also possible to use `this.limit = options.limit || 100000`, which is somewhat shorter.